custom/inddex/reports/nutrient_intakes.py

```diff
@@ -16,18 +18,13 @@ class NutrientIntakesReport(MultiTabularReport, ReportBaseMixin):
     def report_context(self):
         context = super().report_context
         context['export_only'] = self.export_only
-
         return context
 
-    @property
-    def fields(self):
-        return self.get_base_fields()
```

Comment:
note that this is a change in behavior as well - `get_base_fields` didn't include the date field (that was provided by the superclass).  After this change, this report will get that field, which I confirmed with Nick is correct.